typescript/cli/src/commands/relayer.ts

```diff
@@ -0,0 +1,37 @@
+import { HyperlaneCore, HyperlaneRelayer } from '@hyperlane-xyz/sdk';
+
+import { CommandModuleWithContext } from '../context/types.js';
+
+import { agentTargetsCommandOption } from './options.js';
+import { MessageOptionsArgTypes } from './send.js';
+
+export const relayerCommand: CommandModuleWithContext<
+  MessageOptionsArgTypes & { chains?: string }
+> = {
+  command: 'relayer',
+  describe: 'Run a CLI relayer',
+  builder: {
+    chains: {
+      ...agentTargetsCommandOption,
+      demandOption: false,
+    },
```

Comment:
demandOption isn't set on `agentTargetsCommandOption` right? So no need for this more verbose form